ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/ethernet_interfaces.py

```diff
@@ -144,6 +146,35 @@ def ethernet_interfaces(self) -> list | None:
                 context_keys=["name", "peer", "peer_interface"],
             )
 
+            # Adding subinterfaces for each VRF after the main interface.
+            if link["type"] == "underlay_p2p" and "subinterfaces" in link:
```

Comment:
not sure why we compare to `underlay_p2p` should it not be `p2p_vrfs`? (cf line 50/51)

Reply:
actually post refactoring, the link type `p2p_vrfs` does not exist anymore so we may need to decide what we want here